README.adoc

```diff
@@ -1025,6 +1025,32 @@ User.all.size
 User.all.length
 ----
 
+=== Where with Ranges [[where-ranges]]
+
+Use ranges instead of defining comparative conditions using a template for scalar values.
+
+[source,ruby]
+----
+# bad
+User.where("created_at > ?", 30.days.ago).where("created_at < ?", 7.days.ago)
+User.where("created_at > ? AND created_at < ?", 30.days.ago, 7.days.ago)
+User.where("created_at > :start AND created_at < end", start: 30.days.ago, end: 7.days.ago)
+
+# good
+User.where(created_at: 30.days.ago..7.days.ago)
+
+# bad
+User.where("created_at > ?", 7.days.ago)
+
+# good
+User.where(created_at: 7.days.ago..)
```

Comment:
It would be better to mention that Rails 6.0 and later support the syntax.
https://github.com/rubocop/rubocop-rails/issues/689#issuecomment-1115574716

Reply:
Fair enough.

To confirm this. Even though we have this note at the beginning of the guide:
> Some of the advice here is applicable only to recent versions of Rails.

We have the following notes:

> As of Rails 4.2, Active Record supports foreign key constraints natively.

> Since Rails 4.2 YAML configuration files can be easily loaded with the new config_for method

> Rails 4+ enums after attr macros

I'll add a note.

Reply:
Thank you! Fixed.